data/sitesEN.json

```diff
@@ -6424,5 +6424,24 @@
     "destination_main_page": "Main_Page",
     "destination_search_path": "/w/index.php",
     "destination_content_path": "/wiki/"
+  },
+  {
+    "id": "en-bug-fables",
```

Comment:
Please make the ID `en-bugfables` (we only use the hyphen once to separate the language)

Reply:
done